### PR TITLE
Documentation Changes to Demo notebook from Rebased notebook branch

### PR DESCRIPTION
**Pull request by Bytes-Explorer**
## Why are these changes needed?

Documentation changes to the notebook.

### CODE REVIEW

**Comment by blublinsky** [comment]
Is this a replacement for https://github.com/IBM/data-prep-lab/pull/80?

**Comment by shivdeep-singh-ibm** [comment]
Some APIs are based on old codebase. Need to repush the notebook with new apis

**Comment by daw3rd** [comment]
There are more changes pending/merged in PR #106 .

**Review by shivdeep-singh-ibm — APPROVED** [review]
lgtm

**Comment by shivdeep-singh-ibm** [comment]
> There are more changes pending/merged in PR #106 .

updated notebook to include imports from latests paths as updated in#PR106

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+**Prepare virtualenv**
+
+`make venv`
+
+## Launch
+
+2. After dependancies are installed, we can launch the jupyter notebook
+
+`make jupyter`
+
```
Comment (daw3rd) [inline examples/README.md:19]: Can we  suggest a zip file for use? Petros used our repo.  Maybe that  can work?

**Review by daw3rd — CHANGES_REQUESTED** [review]

File: examples/README.md (hunk 1 above)
Comment (Bytes-Explorer) [reply examples/README.md:19]: Yeah, that is a good starting point till we get data approved.

**Review by daw3rd — APPROVED** [review]

**Pull request merged**